cypress/integration/AboutMeAndLocation.feature

```diff
@@ -0,0 +1,49 @@
+Feature: About me and and location
+  As a user
+  I would like to add some about me text and a location
+  So others can get some info about me and my location
+
+  The location and about me are show on the user. Later it will be possible
+  to search for users by location.
+
+  Background:
+    Given I am logged in
+
+  Scenario: I change my name
+    When I visit the "settings" page
+    And I enter "Hansi" as my name
+    And I press "Save"
+    And I can click on my profile picture in the top right corner
+    Then I can see a "Hansi" as my name
+
+    When I refresh the page
+    And I can click on my profile picture in the top right corner
+    Then I can see a "Hansi" as my name
+
+    When I visit the "settings" page
+    And I enter "Peter Lustig" as my name
+    And I press "Save"
+
+  Scenario Outline: I set my location to "<location>"
+    When I visit the "settings" page
+    And I enter "<location>" as my location
+    And I press "Save"
+    And I visit the "profile/peter-lustig" page
+    Then I can see a "<location>" as my location
+
+    Examples: Location
+        | location               |
+        | New York               |
+        | Germany                |
+        | Berlin                 |
+        | Mecklenburg-Vorpommern |
+
+  Scenario: I write about me
+    When I visit the "settings" page
+    And I enter "Some text about me" to about me
+    And I press "Save"
+    And I visit the "profile/peter-lustig" page
```

Comment:
How come that your usename is "Peter Lustig"? You should add that to the scenario.

Reply:
It`s the standard admin user

Reply:
That's correct, but imagine a reader who has never seen our seed data. Therefore you should add the information (e.g. to the cucumber background) that our default user in the database is called "Peter Lustig".